server/controllers/articleController.js

```diff
@@ -156,6 +156,28 @@ const articlesController = {
         analyseRatings(req, res);
       }
     });
+  },
+  /**
+  * @desc validates article input fields
+  * @param  {object} req Http Request object
+  * @param  {object} res Http Response object
+  * @returns {object} httpResponse object
+  */
+  getArticles: (req, res) => {
+    const queryParams = req.paginationQueryParams;
+    // CALCULATE OFFSET
+    const offset = ((queryParams.page - 1) * queryParams.limit);
+    Articles.findAll({
+      offset,
+      limit: queryParams.limit
+    }).then(articles => res.status(200).jsend.success({
+      message: 'Operation Successful',
+      articles,
+      currentPage: queryParams.page
```

Comment:
return a `metadata` object along with this, that will specity:
- total articles
- current page
- articles per page(limit)
- total pages